### ISSUE TITLE
i18next matcher fails when parsing an Array

### ISSUE THREAD

**Issue by apokusin** [leolabs/json-autotranslate#102]
Error when parsing an Array with i18next matcher. This syntax is [supported by i18next ](https://www.i18next.com/translation-function/objects-and-arrays#arrays).

# Steps to reproduce:
1. Add an array in language source JSON `{ "instructions": ["do a thing", "do another thing"] }`
2. Run json-autotranslate `json-autotranslate --source-language en --delete-unused-strings --input locales -m i18next -t key-based -s deepl-free -c ${DEEPL_API_KEY}`

## Expected outcome:
Array of strings is translated

## Actual outcome:
Error
- Using Node v20 but tested in v18 with same results.

```sh
💬 Translating strings from en to de...
├── Translating learn.json (7 changes from cache)
An error has occurred:
input.match is not a function
TypeError: input.match is not a function
    at exports.matchI18Next (/dev/app/node_modules/json-autotranslate/lib/matchers/i18next.js:5:25)
    at exports.replaceInterpolations (/dev/app/node_modules/json-autotranslate/lib/matchers/index.js:20:24)
    at /dev/app/node_modules/json-autotranslate/lib/services/deepl.js:168:47
    at Array.map (<anonymous>)
    at DeepL.<anonymous> (/dev/app/node_modules/json-autotranslate/lib/services/deepl.js:167:43)
    at step (/dev/app/node_modules/json-autotranslate/lib/services/deepl.js:33:23)
    at Object.next (/dev/app/node_modules/json-autotranslate/lib/services/deepl.js:14:53)
    at /dev/app/node_modules/json-autotranslate/lib/services/deepl.js:8:71
    at new Promise (<anonymous>)
    at __awaiter (/dev/app/node_modules/json-autotranslate/lib/services/deepl.js:4:12)

node:child_process:965
    throw err;
    ^

Error: Command failed: json-autotranslate --source-language en --delete-unused-strings --input locales -m i18next -t key-based -s deepl-free -c <API KEY REDACTED>
    at checkExecSyncError (node:child_process:890:11)
    at execSync (node:child_process:962:15)
    at Object.<anonymous> (/dev/app/translate.js:10:1)
    at Module._compile (node:internal/modules/cjs/loader:1376:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1435:10)
    at Module.load (node:internal/modules/cjs/loader:1207:32)
    at Module._load (node:internal/modules/cjs/loader:1023:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:135:12)
    at node:internal/main/run_main_module:28:49 {
  status: 1,
  signal: null,
  output: [ null, null, null ],
  pid: 24320,
  stdout: null,
  stderr: null
}

Node.js v20.10.0
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**Comment by leolabs** [comment]
I just released an update with a new `--with-arrays` flag that you can add if your source files contain arrays. Let me know if this works for you.

**Comment by apokusin** [comment]
Thank you for your speedy response and building this tool, it's already saved us tons of time!

**Comment by leolabs** [comment]
Whoops, it seems that I didn't think this through. The latest version should fix this bug with bfb3723e7034cade3e01ee548d104555425471e4. Does that work on your end as well?

**Comment by apokusin** [comment]
Fixed in `1.22.4`, thank you!
